conmech/features/mesh_features.py

```diff
@@ -38,28 +40,27 @@ def __init__(
             ze_coef,
             density,
             time_step,
-            lambda: self.reorganize_boundaries(is_contact, is_dirichlet),
             False
         )
 
-    def reorganize_boundaries(self, is_contact, is_dirichlet):
+    def reorganize_boundaries(self):
```

Comment:
Passing arguments keeps function easy to test and increases hermatization. It's important especially if we want in the future different boundaries for different effects.